router/core/src/test/scala/io/buoyant/router/ClassifiedTracingTest.scala

```diff
@@ -104,13 +104,4 @@ class ClassifiedTracingTest extends FunSuite with Awaits {
     assert(tracer.iterator.map(_.annotation).toSeq ==
       Seq(Annotation.BinaryAnnotation("l5d.success", 0.84)))
   }
-
-  test("does not trace when is false") {
```

Comment:
I think these tests are not actually correct.

Respect for the `sampled` bit of the trace is a property of the Tracer itself so I don't think it makes sense to test this here.  Indeed, BufferedTracer no longer respects it.